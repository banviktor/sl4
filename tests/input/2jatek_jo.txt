randomization off
questions off
start_game 2
2 3
5 5
inputvector -5 0
jump
start_game 2
exit
