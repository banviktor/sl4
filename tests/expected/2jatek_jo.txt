Véletlenszerû események kikapcsolva.
Kérdezés kikapcsolva.
Pálya betöltve.
Új játék elkezdve 2 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 1.0, 3.0
A robot leesett.
A Sárga Robot nyert!
Pálya betöltve.
Új játék elkezdve 2 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
